api/core/model_runtime/model_providers/zhipuai/text_embedding/embedding-3.yaml

```diff
@@ -0,0 +1,8 @@
+model: embedding-3
+model_type: text-embedding
+model_properties:
+  context_size: 8192
```

Comment:
```suggestion
  context_size: 2048
```

Reply:
https://bigmodel.cn/dev/howuse/model
![image](https://github.com/user-attachments/assets/1fcfb164-3feb-42ca-be78-441fd0a7a886)

I found that the document of zhipu models has been updated. It shows that the context windows is 8k. The default demension is 2k. I'm not sure if I should also change the context window of embedding-2 in this PR. Will it be clearer to split it into two PRs?

Thank you.

Reply:
Yes, please update both context size in this PR. Thank you :)

Reply:
It has been pushed.